src/server/saved_objects/migrations/core/index_migrator.ts

```diff
@@ -78,12 +78,13 @@ export class IndexMigrator {
  * Determines what action the migration system needs to take (none, patch, migrate).
  */
 async function requiredAction(context: Context): Promise<MigrationAction> {
-  const { callCluster, alias, documentMigrator, dest } = context;
+  const { callCluster, alias, documentMigrator, dest, pollInterval } = context;
 
   const hasMigrations = await Index.migrationsUpToDate(
     callCluster,
     alias,
-    documentMigrator.migrationVersion
+    documentMigrator.migrationVersion,
+    pollInterval
```

Comment:
I don't think we should reuse `pollInterval` like this, personally. I don't think the retry delay needs to be configurable personally, but if it is it should get its own config value... or just hard coding it to 1 second.